Networking/NetworkingTests/Remote/WooShippingRemoteTests.swift

```diff
@@ -620,4 +660,19 @@ private extension WooShippingRemoteTests {
                                  defaultAddress: true,
                                  isVerified: true)
     }
+
+    func sampleDestinationAddress() -> WooShippingDestinationAddress {
+        WooShippingDestinationAddress(company: "ACME Corp",
+                                      address1: "789 Oak St",
+                                      address2: "Suite 100",
+                                      city: "Sometown",
+                                      state: "TX",
+                                      postcode: "54321",
+                                      country: "US",
+                                      phone: "12345678901",
```

Comment:
Looks like this just needs to be updated to match the mock data:

```suggestion
                                      phone: "555-0123",
```

Reply:
Done in  https://github.com/woocommerce/woocommerce-ios/pull/15231/commits/ee4663c63e9121e5a39e930fe4c0aa5a8118fc76